.github/workflows/ci.yaml

```diff
@@ -36,6 +36,9 @@ on:
       QUAY_DEPLOYER_PASSWORD:
         required: true
 
+      NGC_API_KEY:
+        required: true
```

Comment:
You will need to pass this through in ci-nightly and ci-on-push as well as ci-devel